modules/cnv_migrate_vm_web.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-migrate-vm-web]]
+== Migrating a virtual machine
```

Comment:
I've dropped it from the assembly so it won't be published in 1.4, but the migrate_vm file is still included in the MR so that the content doesn't get lost. Unless the migration procedure is likely to change for 2.0, in which case I will delete the module completely.

Reply:
It should stay the same pretty much so it is fine to keep it here if the user will not see it.